app/controllers/coronavirus_pages_controller.rb

```diff
@@ -35,8 +35,36 @@ def publish
     end
   end
 
+  def reorder
+    coronavirus_page
+    @old_positions = positions
+    if request.post? && params.key?(:section_order_save)
```

Comment:
Checking if the request is a `POST` and having one action handle both the edit and the update actions is not a standard pattern for rails. I think this could be improved by splitting this into two actions. For a start it would remove the need for the `if` statement completely. I'd even suggest that this could be in a separate reorder controller, which would help in keeping down the size of the `CoronavirusPagesController`.

The `reorder` action (or `CoronavirusPageReordersController` `show` action) could just be:
```ruby
def reorder
  coronavirus
end
```
Then an `update_reorder` action could contain lines 40 + 42 to 51, and the methods called.